jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -426,47 +500,20 @@ ecma_builtin_try_to_instantiate_property (ecma_object_t *object_p, /**< object *
 
   uint32_t index = (uint32_t) (curr_property_p - property_list_p);
 
-  JERRY_ASSERT (index < 64);
+  JERRY_ASSERT (index < 96);
```

Comment:
Actually this could be extended to 256 at some point, but I want to start with this limit. Each of these uses can be extended further by modifying this code.